webflow-hacks/misc-hacks/37/37.ts

```diff
@@ -0,0 +1,16 @@
+window.Webflow ||= [];
+window.Webflow.push(() => {
+  const tabLinks = document.querySelectorAll<HTMLAnchorElement>('.w-tab-link');
+  if (tabLinks.length === 0) return;
+
+  // add listeners and click chosen tab
+  const chosenTabId = localStorage.getItem('tab');
```

Comment:
Can we use a more specific `localStorage` key? Something like `previousTabId` instead of `tab`.
`tab` feels too much generic and could potentially collide with other libraries/solutions existing on the page.
Also, can we create a constant for the key? This way it's easier to identify and update if needed.